### PR TITLE
Apprenda 6.0 Small Cluster

### PR DESCRIPTION
**Pull request by dutronlabs**
The small cluster template provides the ability for Azure customers to provision a cross-platform-supported environment, within the confines of the Apprenda Express license. This allows for both .NET and generic linux (ie. Java, Docker) applications to run.

Large cluster is incoming, need to finalize how the license is on-boarded during provisioning (otherwise, it's the same template, just larger).

### CODE REVIEW

File: apprenda60-small/metadata.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Apprenda 5.5 Single Node",
```
Comment (singhkay) [inline apprenda60-small/metadata.json]: Does this need to be updated?

File: apprenda60-small/metadata.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  "description": "This is the express version of Apprenda, with a single Windows VM.",
+  "summary": "Apprenda 5.5 Express",
```
Comment (singhkay) [inline apprenda60-small/metadata.json]: This as well?

**Comment by singhkay** [comment]
@dutronlabs Template looks good but you might want to update the metadata.json to reflect Apprenda 6.0

**Comment by dutronlabs** [comment]
should be updated for 6.0, also has some changes in here that represent fixes based on feedback from Kundana.

**Comment by singhkay** [comment]
@dutronlabs Looks good, merged

**Pull request merged**